tcr-engine/toolchain/command.go

```diff
@@ -102,14 +102,16 @@ func (command Command) runsWithArch(arch ArchName) bool {
 	return false
 }
 
-func (command Command) run() error {
+func (command Command) run() (string, error) {
```

Comment:
Minor improvement: using named return variables (output string, err error) could allow to simplify slightly the body of this method